.eslintrc

```diff
@@ -5,13 +5,10 @@ ecmaFeatures:
   - generators
 env:
   node: true
+  es6: true
```

Comment:
Most of these changes look fine, but CI is heavily broken due to many of them. I would like one of three things to happen: this PR depends on one which adjusts `.eslintrc` and cleans up those failures, this PR cleans up those failures itself, or the bulk of the `.eslintrc` changes are left out for now and addressed later.